vispy/app/backends/_qt.py

```diff
@@ -590,7 +590,6 @@ def _vispy_set_current(self):
         if self._vispy_canvas is None:
             return  # todo: can we get rid of this now?
         if self.isValid():
-            self._vispy_canvas.set_current()  # Mark as current
```

Comment:
Ahh, I see the related change below that does it instead. I think the problem is that there might be places in these backends where we call `self._vispy_set_current()`, those will need to be changed to `self._vispy_canvas.set_current()` instead.